client/src/redux/ga-saga.js

```diff
@@ -23,14 +23,19 @@ function* callGaType({ payload: { type, data } }) {
       const email = yield select(emailSelector);
       const completedChallengeTotal = yield select(completedChallengesSelector);
       const completedChallengeSession = yield select(completionCountSelector);
-      const customDimensions = {
-        Test_Variation: emailToABVariant(email).isAVariant ? 'A' : 'B',
-        Test_Type: aBTestConfig.type,
-        Challenges_Completed_Session: completedChallengeSession,
-        Challenges_Completed_Total: completedChallengeTotal.length,
-        URL: window.location.href
+      const CustomDimensions = {
```

Comment:
There is nothing special about this object from a JS point of view. So, I suggest sticking with naming convention

```suggestion
      const customDimensions = {
```